### PR TITLE
Add sprint report 35

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: _posts/development/2018-04-05-sprint-report-35.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Features
+
+## Diffing of superseded requests
+OBS makes it tremendously easy to fix bugs in packages and submit it back to the distribution :muscle:.
+However, to get it back into the distribution, your submit request needs to pass the openSUSE review team.
+Often it happens that you get important feedback on your submit request and need to update it.
+In OBS, this happens with creating a new submit request which supersedes the old one.
+
+In the past it was necessary to switch back and forth between the superseded and superseding request and check if all suggestions got added correctly. Exhausting :sweat:!
+Therefore a while ago, [Ludwig Nussel](https://github.com/openSUSE/open-build-service/issues/3915), requested a feature to make it easier to review these superseded requests which we finally implemented.
```
Comment (Ana06) [inline _posts/development/2018-04-05-sprint-report-35.md]: I think the comas are strange in this sentence... :thinking:

File: _posts/development/2018-04-05-sprint-report-35.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+When you go now to a superseded request, you get a message that this request got superseded and if you want to see the diff of the superseded and superseding request instead.
```
Comment (Ana06) [inline _posts/development/2018-04-05-sprint-report-35.md]: and if you want to -> you can 

if not the message should ask you if you want to see...

File: _posts/development/2018-04-05-sprint-report-35.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Check out this [pull request](https://github.com/openSUSE/open-build-service/pull/4741) for more information.
+
+![image](/images/posts/sprint_35_request_1.png)
```
Comment (Ana06) [inline _posts/development/2018-04-05-sprint-report-35.md]: should we mention something about the broken javascript? maybe pointing here: https://github.com/openSUSE/open-build-service/issues/4719

File: _posts/development/2018-04-05-sprint-report-35.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+As you might already have [heard](http://openbuildservice.org/2018/03/01/cloud-upload/), OBS is now able to upload built EC2 images to Amazon Web Service :cloud:.
+This sprint we added API routes for
+
+* Starting a cloud upload
+* Aborting cloud uploads
+* Listing all existing cloud uploads (succeeded, failed and aborted ones)
+
+This allows you to automate your workflow :smiley:
+
+# Bug fixes
```
Comment (Ana06) [inline _posts/development/2018-04-05-sprint-report-35.md]: It seems I forgot to add this https://trello.com/c/wN3ncTZe/702-p4-fix-request-review-view-for-role-requests in the sprint report :see_no_evil: 

@ChrisBr I'll send you something about it in a moment. :wink:  So please wait for that before merging. :confounded:

File: _posts/development/2018-04-05-sprint-report-35.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+## Enforce version format validation in KIWI editor
+ 
+Kiwi expects image versions to be in a certain format (3 numbers split by dots), otherwise an image build might fail. 
+We now added a validation to the kiwi editor.
+ 
+# Releases
+
+## Release of OBS 2.9
+
+Last week we [released](http://openbuildservice.org/2018/03/19/release_of_obs_2.9/) OBS 2.9. :tada: Read the linked blog post to learn more about the cool features it brings. Like the kiwi editor, Ec2 cloud upload and RabbitMQ support :green_heart:
```
Comment (Ana06) [inline _posts/development/2018-04-05-sprint-report-35.md]: I do not like to say the linked blog post. I would instead say the blog post and link it again. Not that important though. :thinking:

File: _posts/development/2018-04-05-sprint-report-35.md (hunk 3 above)
Comment (Ana06) [reply _posts/development/2018-04-05-sprint-report-35.md]: instead of image you can use a more helpful name :wink:

**Comment by Ana06** [comment]
@ChrisBr here is the part about the request review view, probably for the bug section:

## Fix request review view for role requests

When reviewing a request of type `set_bugowner` and `add_role` there was some information missing in the request review page, such as the user or group for which we were requesting the role, and in some cases, there was no information at all. That made really complicate the request role revision. How should I know if I want to accept a request if I do not know who will get the role? :confused: This have been fixed and now all the information is shown. We have even made clearer the difference between `add_role bugowner` and `set_bugowner`: :tada: 

set_bugowner


add_role bugowner

File: _posts/development/2018-04-05-sprint-report-35.md (hunk 3 above)
Comment (ChrisBr) [reply _posts/development/2018-04-05-sprint-report-35.md]: The javascript is unrelated, I just discovered it while testing it... Most requests in webui are anyway single requests

**Comment by ChrisBr** [comment]
@Ana06 updated

**Pull request merged**